report_py3o/models/_py3o_parser_context.py

```diff
@@ -32,6 +32,8 @@ def __init__(self, env):
         self._env = env
 
         self.localcontext = {
+            'user': self._env.user,
+            'lane': self._env.lang,
```

Comment:
s/lane/lang/

Reply:
done. Thank you

Reply:
We should tests all the methods provided by the parser context into the demo template to avoid regressions and also tests this part of code...